lib/http-bridge/test/integration/Main.hs

```diff
@@ -96,24 +98,35 @@ main = do
         let stateDir = "./test/data/cardano-node-simple"
         let networkDir = "/tmp/cardano-http-bridge/networks"
         let bridgePort = 8080
+        let nodeApiAddress = "127.0.0.1:3101"
         removePathForcibly (networkDir <> "/local")
         handle <-
             openFile "/tmp/cardano-wallet-launcher" WriteMode
-        systemStart <-
-            formatTime defaultTimeLocale "%s" . addUTCTime 5 <$> getCurrentTime
-        cluster <- async $ void $ launch
-            [ cardanoNodeSimple stateDir systemStart ("core0", "127.0.0.1:3000")
-            , cardanoNodeSimple stateDir systemStart ("core1", "127.0.0.1:3001")
-            , cardanoNodeSimple stateDir systemStart ("core2", "127.0.0.1:3002")
-            , cardanoNodeSimple stateDir systemStart ("relay", "127.0.0.1:3100")
-            , cardanoHttpBridge bridgePort"local" networkDir handle
+        start <-
+            formatTime defaultTimeLocale "%s" . addUTCTime 2 <$> getCurrentTime
+        [h0, h1, h2, h3] <- forM ["core0", "core1", "core2", "relay"] $ \x -> do
+            openFile ("/tmp/cardano-node-simple/" <> x) WriteMode
```

Comment:
cf comment below, we have to redirect stderr & stdout _somewhere_ and can't just use `NoStream` otherwise the node fails to ... fail and the processes never terminate on failure.